### PR TITLE
create haphpipe recipe

### PR DESCRIPTION
**Pull request by maggiesteiner**
Create recipe for HAPHPIPE v1.0.0.

### CODE REVIEW

**Comment by maggiesteiner** [comment]
@BiocondaBot please add label

File: recipes/haphpipe/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    - raxml 
+    - sra-tools
+
+test:
+  commands:
+    - haphpipe -h
+
+about:
+  home: https://github.com/gwcbi/haphpipe
+  license: 'open source'
```
Comment (dpryan79) [inline recipes/haphpipe/meta.yaml:50]: Can you clarify what sort of "open source" license this has? Ideally it's something like MIT or GPL, but technically we need to know (and they probably need to put a copy of the license together with the source code).

Comment (maggiesteiner) [reply recipes/haphpipe/meta.yaml:50]: Thank you for reviewing this! I am following up with my PI now about the license. In the meantime, could you please explain the changes made in your recent commits, particularly in removing the host requirements?

Comment (dpryan79) [reply recipes/haphpipe/meta.yaml:50]: The host requirements aren't actually needed for installation, just for running things, so my changes just speed up building a bit.

Comment (dpryan79) [reply recipes/haphpipe/meta.yaml:50]: BTW, thanks for the PR!

Comment (dpryan79) [reply recipes/haphpipe/meta.yaml:50]: The changes to `python=3.7` are just to future proof things a bit. We're likely to start supporting python 3.8 within the next month or so and I'd prefer to not have to modify so many recipes that artificially restrict python compatibility to specific ranges.

Comment (maggiesteiner) [reply recipes/haphpipe/meta.yaml:50]: Thank you for the clarifications! 

We'd like to use the MIT License.

Comment (maggiesteiner) [reply recipes/haphpipe/meta.yaml]: @dpryan79 We have addressed the License issue and updated our repository accordingly. I have committed and tested the updated recipe now!

**Comment by maggiesteiner** [comment]
@BiocondaBot please add label

**Comment by dpryan79** [comment]
@bioconda-bot please fetch artifacts

**Comment by dpryan79** [comment]
@bioconda-bot please merge

Thanks @maggiesteiner !

**Pull request merged**